assets/lokomotive-kubernetes/bootkube/resources/charts/kubernetes/templates/kube-controller-manager.yaml

```diff
@@ -60,6 +60,8 @@ spec:
         - --cluster-signing-cert-file=/etc/kubernetes/secrets/ca.crt
         - --cluster-signing-key-file=/etc/kubernetes/secrets/ca.key
         - --configure-cloud-routes=false
+        - --controllers=*,tokencleaner
+        - --experimental-cluster-signing-duration=1h
```

Comment:
This is still an issue of discussion. What should be the duration by default here?

Reply:
Do we have any strong reason **not to use** upstream default?

Reply:
Upstream default is 1yr which is equal to the default expiry duration of the root CA.

Reply:
IMO it's fine like this, as we still have a benefit of using TLS bootstrapping, as it allows to enable node restrictions and each node use unique certificate.